ddtrace/bootstrap/sitecustomize.py

```diff
@@ -127,6 +127,10 @@ def update_patched_modules():
     # are initialized
     if asbool(os.getenv("DD_INSTRUMENTATION_TELEMETRY_ENABLED")):
         telemetry_writer.enable()
+    else:
+        # TelemetryWriter._enabled is set to None by default which allows integrations to be queued at start up
+        # TelemetryWriter.disable flushes telemetry data and disables this feature
+        telemetry_writer.disable()
```

Comment:
do we need to call this for customers not using `ddtrace-run`?

Reply:
Hmm, if the writer is never disabled, integrations will just get added to a queue that will never be freed.
I would expect that because there is a fairly limited number of them it won't be a problem.

Reply:
I'll remove the else branch